### PR TITLE
Ingestion server Docker improvements

### PR DESCRIPTION
**Pull request by zackkrida**
- Switch from `creativecommons` to `openverse` docker hub account
- Add documentation about deploying the ingestion server to docker in the ingestion server README

### CODE REVIEW

File: ingestion_server/README.md [hunk 1]
```diff
@@ -116,3 +116,11 @@ database_table_to_elasticsearch_model = {
...
 }
 ```
+
+# Deployment (last deployed version: 1.20.0)
+
+This codebase is deployed as Docker image to Docker hub. The deployed image is then pulled in the production environment. See the [`./publish_release.sh`](publish_release.sh) script for deploying to Docker hub. The machine running the script must be logged into Docker as the Openverse account.
+
+The current docker hub image is at [openverse/ingestion_server](https://hub.docker.com/r/openverse/ingestion_server) but the image previously lived at [creativecommons/ingestion_server](https://hub.docker.com/r/creativecommons/ingestion_server).
+
+The version numbering for the Docker image appears to be a standalone semver value, seperate from the repository version.
```
Comment (krysal) [inline ingestion_server/README.md:126]: A little typo here: "... _separate_ from the repository version"

**Review by krysal — COMMENTED** [review]
It's looking good!

File: ingestion_server/README.md [hunk 2]
```diff
@@ -116,3 +116,11 @@ database_table_to_elasticsearch_model = {
     'image': Image
 }
 ```
+
+# Deployment (last deployed version: 1.20.0)
+
+This codebase is deployed as Docker image to Docker hub. The deployed image is then pulled in the production environment. See the [`./publish_release.sh`](publish_release.sh) script for deploying to Docker hub. The machine running the script must be logged into Docker as the Openverse account.
+
+The current docker hub image is at [openverse/ingestion_server](https://hub.docker.com/r/openverse/ingestion_server) but the image previously lived at [creativecommons/ingestion_server](https://hub.docker.com/r/creativecommons/ingestion_server).
```
Comment (dhruvkb) [inline ingestion_server/README.md:124]: Along the same lines, 
```suggestion
The current Docker hub image is at [openverse/ingestion_server](https://hub.docker.com/r/openverse/ingestion_server) but the image previously lived at [creativecommons/ingestion_server](https://hub.docker.com/r/creativecommons/ingestion_server).
```

**Review by obulat — APPROVED** [review]
Thank you for adding documentation! I didn't realize that the Docker images were pulled from Docker hub for deployment.

File: ingestion_server/README.md (hunk 1 above)
Comment (zackkrida) [reply ingestion_server/README.md:126]:
```suggestion
The version numbering for the Docker image appears to be a standalone semver value, separate from the repository version.
```

**Pull request merged**